#Goal
string group_name
string fjt_action_name
string[] joints_names
float64[] target_joints_configurations
int16 scaling
bool simulation
string speed_scaling_topic
string pipeline_id
string planner_id
float64 acceleration_scaling_factor
float64 velocity_scaling_factor
---
#Result
bool ok
string error
---
#Feedback

